.pyup.yml

```diff
@@ -0,0 +1,6 @@
+schedule: "every day"
+search: False
+update: insecure
```

Comment:
That's for security updates only: https://pyup.io/docs/configuration/#control-whats-being-updated